### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by mkinney**
Related to #193 

Add spellcheck to flake8

It is kinda non-intelligent spell checking (does not strip numbers/quotes), but I think having it might better than not.

I think it caught some bugs (see the deafult fixes below).

Thoughts?

### CODE REVIEW

**Comment by mkinney** [comment]
Hmm... python 2.7 is dead. Do we care about it anymore?

File: setup.py [hunk 1]
```diff
@@ -42,6 +42,7 @@
     'flake8',
     'flake8-commas',
     'flake8-quotes',
+    'flake8-spellcheck',
```
Comment (Fizzadar) [inline setup.py]: To fix py2 we could just turn this into a list and append this requirement only if py3.

Comment (Fizzadar) [reply setup.py]: Actually - better than that - we can just modify this line (discovered via https://stackoverflow.com/questions/34407802/how-can-i-specify-a-requirement-only-for-python-2-ie-not-python-3-in-requireme):

```py
    'flake8-spellcheck ; python_version >= "3"',
```

Comment (Fizzadar) [reply setup.py]: Realised I could just do it as such a small change; hope you don't mind! https://github.com/Fizzadar/pyinfra/pull/217/commits/fb96848867e269c36e220a98d8717f444486f71a

Comment (mkinney) [reply setup.py]: No worries!

Thanks

**Pull request merged**